packages/sdk/src/wallet/Ordit.ts

```diff
@@ -98,17 +98,12 @@ export class Ordit {
     this.#network = value
   }
 
-  getAddressByType(type: AddressFormats) {
+  getAddressByType(type: AddressFormats, accountIndex: number, addressIndex: number) {
```

Comment:
let's make this param be named so we have to pass object with the param names explicitly to be clearer since we are handling 2 index

Reply:
Updated